client/src/pages/dashboard/Dashboard.css

```diff
@@ -35,6 +28,108 @@
   }
 
 .proposal-description {
-    width: 50%;
+    width: 40%;
```

Comment:
The centering of the proposal-description looks slightly off (the space on the right is much greater than the space on the left)